### DISCUSSION TITLE
Property variables compatible with link format

### DISCUSSION THREAD

**Discussion by Moyf** [chrisgurney/obsidian-note-toolbar#73] [Q&A] [answered]
Hi I tried to use the property variable to open other note, for example, I added `prev: My-Note` in the property, and use `{{prop_prev}}` to open the note, it works great.
Meanwhile, some other Plugin would only recognize the link format ( `prev: [[My-Note]]`), so just wondering is it okay to also open note with this kind of link format?

**Comment by chrisgurney** [comment]
@Moyf Yup both link formats do work! 

I just tested with 1.7 to re-confirm. Let me know if it doesn't work for you.

**Reply by Moyf** [reply]
Thanks for replying!

~~Just re-check it, and I guess it might be related to the Language~~ 🤔
I use Chinese in file name and it would warn:
![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/4013062/03f9cc02-24da-48cf-bc98-7e3166964428)

well, even with pure English name it would also warn this:
![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/4013062/ea7aa2c5-48c2-4856-a6a8-885750cc6a12)

---

Oh, I see!

When I input the link, I selected the alias:
![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/4013062/a2ec470c-331e-48c3-83f6-760c79671cc0)

And it would be stored as `[[UC|UC]]`, which would cause the warning 😂
![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/4013062/0a75bcbe-4679-4772-a830-32b7e8f50fe0)

When clicking it directly, OB would know to only extract the front part I guess.

**Reply by chrisgurney** [reply]
@Moyf Good catch! 

I'm not sure if it's exactly the same problem, but I've logged a bug here:
https://github.com/chrisgurney/obsidian-note-toolbar/issues/75

Do you have an example of a link that works, with an alias, in a standard note, but does not work in a toolbar?

If I fix the above issue, do you think it will solve your problem?

**Reply by Moyf** [reply]
Yay I think if you resolve the link(with Aliases) with only the part before "|" then the problem would be fixed!

And my guess is that maybe it's the OS (I'm using Windows, which do not allow `|` in file name) causing the different behavior when clicking the button 
![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/4013062/08d3d27b-a98f-4995-894a-b7e439b89c38)

**Reply by chrisgurney** [reply]
@Moyf Okay good to hear!

**Comment by chrisgurney** [comment, marked as answer]
The problem actually identified was moved to issue #75 which should now be fixed in v1.8.2
